TrustSDK.xcodeproj/project.pbxproj

```diff
@@ -765,6 +765,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_BITCODE = NO;
```

Comment:
If this only disabled for the current framework?

Reply:
This needs to be disabled on all other pod as well for an one time build to have a .framework which is unoptimized to bind on it in a separate binding project.